### ISSUE TITLE
Extra exp in softmax formula?

### ISSUE THREAD

**Issue by shirriff** [tensorflow/tensorflow#188]
The Vector Representations of Words tutorial says:
P(wt|h)=softmax(exp{score(wt,h)}) =exp{score(wt,h)} / ∑exp{score(w′,h)}.

Shouldn't that be ... softmax(score(wt,h)) ... since softmax applies an exp to its arguments?

Page is http://www.tensorflow.org/tutorials/word2vec/index.md

**Comment by gouwsmeister** [comment]
Yes, thanks for reporting this. I sent out a fix; it should be updated pretty soon.

**Comment by vrv** [comment]
Fixed and pushed in https://github.com/tensorflow/tensorflow/commit/a9ca5173b2252b0de5dd754147b275e85298e522
